pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm

```diff
@@ -183,13 +190,12 @@ sub _suggest_imunify360 {
             block_notify => 1,                                                                                                      # Do not send a notification about this
         );
     }
-    elsif ( !Whostmgr::Imunify360::is_imunify360_installed() ) {
+    elsif ( $licensed && !$installed ) {
```

Comment:
This one adds `$licensed &&` but it did not have `Whostmgr::Imunify360::is_imunify360_licensed() &&` before. Was this on purpose?

Reply:
I think it was probably added intentionally to be explicit, but logically it's not required. If you look at the surrounding `if`/`elsif` blocks, it is logically impossible to reach this `elsif` except in the case where `$licensed` is true.

If you would prefer to have it removed to avoid confusion, I can do that.

Reply:
nah, as long as there was a reason ;)